sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/ModelProperties.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Azure.DigitalTwins.Core.Serialization
+{
+    /// <summary>
+    /// Properties on a digital twin that adhere to a specific model.
+    /// </summary>
+    public class ModelProperties
+    {
+        /// <summary>
+        /// Information about the model a digital twin conforms to. This field is present on every digital twin.
+        /// </summary>
+        [JsonPropertyName("$metadata")]
+        public DigitalTwinMetadata Metadata { get; set; } = new DigitalTwinMetadata();
```

Comment:
Component also uses DigitalTwinMetadata  type? It seems ok but wondering if we should ask DT team.

Reply:
These are data types I dreamed up by looking at example digital twin output (like the swagger [examples](https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk/digitaltwins/Azure.DigitalTwins.Core/src/swagger/examples/GetTwin.json)).

The important part is that a pattern repeats twice. I figured I'd reuse a class to handle that repetition.

Reply:
I see sounds good :)